test/sqllogictest/system-cluster.slt

```diff
@@ -99,6 +99,10 @@ Explained Query (fast path):
 Used Indexes:
   - mz_internal.mz_show_materialized_views_ind (*** full scan ***)
 
+Notices:
+  - Notice: Index mz_internal.mz_show_materialized_views_ind on mz_show_materialized_views(schema_id, cluster_id) is too wide to use for literal equalities `schema_id = "u3"`.
+    Hint: If your literal equalities filter out many rows, create an index whose key exactly matches your literal equalities: (schema_id).
```

Comment:
Well, this definitely contributes to the ongoing discussion of whether to show notices by default, but I'm not sure if it's pro or against...

Reply:
I changed the keys of builtin indexes `mz_show_materialized_views_ind` and `mz_show_indexes_ind`, so that SHOW MATERIALIZED VIEWS and SHOW INDEXES with no arguments are able to do a lookup, and thus they don't show a notice that the index is too wide.